storage/memory/memory.go

```diff
@@ -279,7 +282,7 @@ func (c *checker) CheckLimitAndUpdate() bool {
 	}
```

Comment:
Please also update the comment of this function to account for the change in the return value

Reply:
Done